app/services/db/sp_return_log.rb

```diff
@@ -11,7 +12,15 @@ def self.create_return(request_id:, user_id:, billable:, ial:, issuer:, requeste
         returned_at: Time.zone.now,
       )
     rescue ActiveRecord::RecordNotUnique
+      # Can be removed after deploy of RC 185
```

Comment:
I don't know that we have much of a problem cleaning these sorts of things up, but it occurs to me it'd be neat to have a lint which fails on a branch that tries to merge to `stages/prod` if there's specific code marked to have been removed by the release (using consistent formatting like `# REMOVEME: RC 186`).

I'd done something similar in a past project: https://github.com/WordPress/gutenberg/pull/5491

Reply:
For sure, I'd be open to something like https://github.com/searls/todo_or_die